modules/svg/geolocate.js

```diff
@@ -0,0 +1,147 @@
+import { select as d3_select } from 'd3-selection';
+
+import { svgPointTransform } from './helpers';
+import { geoMetersToLat } from '../geo';
+import {
+    geoIdentity as d3_geoIdentity,
+    geoPath as d3_geoPath,
+    geoStream as d3_geoStream
+} from 'd3-geo';
+import _throttle from 'lodash-es/throttle';
+
+export function svgGeolocate(projection, context, dispatch) {
+    var throttledRedraw = _throttle(function () { dispatch.call('change'); }, 1000);
```

Comment:
Not 100% sure but I don't *think* this code should ever need to dispatch `change`.. This forces everything to redraw.  Maybe this code could maintain its own timer to refresh the location circle sometimes.